File: src/tasks/cli/arg/InfoArguments.java
package tasks.cli.arg;

import omnia.data.cache.Memoized;
import omnia.data.structure.List;
import tasks.model.Task;
import tasks.model.TaskStore;

public final class InfoArguments extends SimpleArguments {
  private InfoArguments(List<Task> tasks) {
    super(tasks);
  }

  static final class Parser extends SimpleArguments.Parser<InfoArguments> {
    protected Parser(Memoized<TaskStore> taskStore) {
      super(taskStore, InfoArguments::new);
    }
  }
}

File: cli/src/me/strangepan/tasks/cli/feature/info/InfoArguments.kt
package me.strangepan.tasks.cli.feature.info

import omnia.data.structure.List
import me.strangepan.tasks.cli.command.common.simple.SimpleArguments
import me.strangepan.tasks.engine.model.Task

/** Model for parsed Info command arguments.  */
class InfoArguments internal constructor(tasks: List<Task>) : SimpleArguments(tasks)